quic/s2n-quic-transport/src/stream/stream_container.rs

```diff
@@ -282,14 +280,14 @@ macro_rules! iterate_uninterruptible {
                 mut_stream.get_stream_interests()
             };
 
-            did_finalize |= $sel.interest_lists.update_interests(
+            $sel.interest_lists.update_interests(
```

Comment:
was this causing an issue, or its just a better way to do it?

Reply:
Just a better way to do it. Since it's just a bool return value it's possible that it gets ignored and then the stream may stick around longer than it should.